pubsub/google/cloud/pubsub_v1/publisher/_batch/thread.py

```diff
@@ -75,6 +75,9 @@ def __init__(self, client, topic, settings, autocommit=True):
         # These members are all communicated between threads; ensure that
         # any writes to them use the "state lock" to remain atomic.
         self._futures = []
+        # _futures list should remain unchanged after batch
```

Comment:
Please move the new comment above line 77. Comments should be above the code they explain.

Reply:
Done